waypoints:
  - name: point1
    pose:
      position:
        x: 0.0
        y: 0.0
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
    work_type: 0
    action_group:
      - action_content:
          light_open: false
          body_up: false
          delay_time: 0
          rotation: 0.0
          picture: 0
          head_rpy: 
            pitch: 0.0
            roll: 0.0
            yaw: 0.0

